src/Rules/Spryker/DynamicMethodMissingPhpDocAnnotationRule.php

```diff
@@ -29,17 +29,14 @@ class DynamicMethodMissingPhpDocAnnotationRule implements Rule
 
     /**
      * @param string $className
-     * @param array $methodNames
+     * @param string[] $methodNames
      */
     public function __construct(string $className, array $methodNames)
     {
         $this->className = $className;
         $this->methodNames = $methodNames;
     }
 
-    /**
-     * @return string
-     */
```

Comment:
Why removing those?

Reply:
That's how the issue `Return type (string) of method                                         
         PHPStan\Rules\Spryker\DynamicMethodMissingPhpDocAnnotationRule::getNo  
         deType() should be covariant with return type                          
         (class-string<PhpParser\Node>) of method                               
         PHPStan\Rules\Rule<PhpParser\Node>::getNodeType()` is fixed. You could check the similar rule with this fix [here](https://github.com/phpstan/phpstan-src/blob/eeae2da7999b2e8b7b04542c6175d46f80c6d0b9/src/Rules/Comparison/ImpossibleCheckTypeMethodCallRule.php#L32).

Reply:
Makes no sense ;) Since this is still a string. just a class string.
For that one usually has special annotations on top. But removal should not be necessary IMO.

I wonder if our sniffer is OK with this.